tests/entrypoints/openai/test_chat_template.py

```diff
@@ -85,10 +97,10 @@ def test_no_load_chat_template_literallike():
 
 
 @pytest.mark.parametrize(
-    "model,template,add_generation_prompt,continue_final_message,expected_output",
+    "model,template,add_generation_prompt,continue_final_message,echo,expected_output",
```

Comment:
Looks like some entries don't follow this format. To avoid similar mistakes in the future, can you use `typing.NamedTuple` objects in `MODEL_TEMPLATE_GENERATON_OUTPUT`?

Reply:
Hi, @DarkLight1337 

I found that the `test_get_gen_prompt` function doesn't seem to test the `echo` parameter because it calls the `apply_hf_chat_template` function, which doesn't support the `echo` parameter.

It seems that only by testing the `create_chat_completion` function can the `echo` parameter be tested.

I need some time to figure out how to test the `create_chat_completion` function.

Reply:
You can take a look at the OpenAI entrypoints tests for some examples.